docs/macros_schema.md

```diff
@@ -0,0 +1,21 @@
+{% docs ref %}
+    This macro is the default dbt macro  for the model. ref is used in the sql file to refer to the model name.
+
+    example:
+    
+        ref('stg_payments') -- where stg_payments is the ref model name
+        or
+        ref('stg_payments', true) -- where raw_customers is the source model and true is to include the database name
+
+{% enddocs %}
+
+{% docs src %}
+    This macro is the default dbt macro  for the model. source is used in the sql file to refer to the source model name.
```

Comment:
This macro overrides the default 'source' dbt macro when in your local dbt project. This is used in your models to reference source models

Reply:
done